docs/api/core/Object3D.html

```diff
@@ -280,6 +280,41 @@
 		Searches through the object's children and returns the first with a matching id.
 		</div>
 
+		<h3>[method:Vector3 getWorldPosition]([page:Vector3 vector])</h3>
+		<div>
+		vector — (optional)<br />
```

Comment:
<h3>[method:Vector3 getWorldPosition]([page:Vector3 optionalTarget])</h3>
    <div>
    optionalTarget — Optional target to set the result. Otherwise, a new `Vector3` is instantiated. (optional)<br />